config/github-labels.yml

```diff
@@ -0,0 +1,152 @@
+remove:
+  - greenkeeper
+  - has pr
+  - needs pr
+  - priority
+  - low priority
+# List of labels to add.
+# Can be groups, as that allows color scaling (or defining a color once)
+# Each label can have a list of `legacy` labels, which are replaced.
+add:
+  - color:
+      from: hsl(212, 97%, 58%)
+      to: hsl(212, 97%, 88%)
+    labels:
+      - name: 📦 area/dependencies
+        description: This affects dependencies
+        legacy:
+          - dependencies # MDXs old labels
+      - name: 📚 area/docs
+        description: This affects documentation
+        legacy:
+          - docs # MDXs old labels
+      - name: 👀 area/external
+        description: This affects something outside of the project
+        legacy:
+          - external # wooorm’s old labels
+          - not in core # wooorm’s old labels
+      - name: 🗄 area/interface
+        description: This affects the public interface
+      - name: 🏁 area/performance
+        description: This affects performance
+      - name: 🔒 area/security
+        description: This affects security
+      - name: 🕸️ area/test
+        description: This affects tests
+        legacy:
+          - reproduction # MDXs
+      - name: 🏗 area/tools
+        description: This affects tooling
+      - name: ☂️ area/types
+        description: This affects typings
+  - color:
+      from: hsl(0, 97%, 58%)
+      to: hsl(0, 97%, 88%)
+    labels:
+      - name: 🌊 blocked/upstream
+        description: This cannot progress before something external happens first
+        legacy:
+          - upstream # MDXs
+      - name: 🗳 blocked/vote
+        description: This cannot progress before voting is complete
+      - name: 🛠 blocked/wip
+        description: This cannot progress yet, it’s being worked on
+  - color:
+      from: hsl(45, 97%, 58%)
+      to: hsl(45, 97%, 88%)
+    labels:
+      - name: 👶 semver/patch
+        description: This is a backwards-compatible fix
+        legacy:
+          - patch # wooorm’s old labels
+      - name: 🧒 semver/minor
+        description: This is backwards-compatible change
+        legacy:
+          - feature # wooorm’s old labels
+          - minor # wooorm’s old labels
+      - name: 🧑 semver/major
+        description: This is a change
+        legacy:
+          - change # wooorm’s old labels
+          - major # wooorm’s old labels
+  - color:
+      from: hsl(300, 97%, 58%)
+      to: hsl(300, 97%, 88%)
+    labels:
+      - name: 🙆 status/confirmed
+        description: 'This is open and confirmed: ready to be worked on'
+        legacy:
+          - rfc accepted # MDXs old labels
+          - investigation # MDXs old labels
+      - name: 🥂 status/merged
+        description: This is done!
+      - name: 🔍 status/open
+        description: This is new!
+      - name: ⛵️ status/released
+        description: This is released
+      - name: 🧘 status/waiting
+        description: This may go somewhere but needs more information
+        legacy:
```

Comment:
what does legacy do?

Reply:
> Each label can have a list of `legacy` labels, which are replaced.

Legacy is used primarily so that, when a legacy label is used, it’s renamed (or relabelled).
That means that the old labels still work (e.g., issues with enhancement previously, now have the new `type/enhancement` label).